test/e2e/scheduling.go

```diff
@@ -74,12 +72,7 @@ var _ = Describe("ReplicaSchedulingPreferences", func() {
 				tl.Fatalf("Error initializing dynamic client: %v", err)
 			}
 			for targetTypeName := range schedulingTypes {
-				typeConfig := &fedv1a1.FederatedTypeConfig{}
-				key := client.ObjectKey{
-					Namespace: f.FederationSystemNamespace(),
-					Name:      targetTypeName,
-				}
-				err = dynClient.Get(context.Background(), key, typeConfig)
+				typeConfig, err := common.GetTypeConfig(dynClient, targetTypeName, f.FederationSystemNamespace())
```

Comment:
Please rebase the code. This file is changed and need to update related code based on your pr.

Reply:
Done!